bootstrap/kubeadm/internal/ignition/clc/clc_test.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package clc_test tests clc package.
+package clc_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	bootstrapv1 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha4"
+	"sigs.k8s.io/cluster-api/bootstrap/kubeadm/internal/cloudinit"
+	"sigs.k8s.io/cluster-api/bootstrap/kubeadm/internal/ignition/clc"
+)
+
+const configWithWarning = `---
+storage:
+  files:
+  - path: /foo
+    contents:
+      inline: foo
+`
+
+func Test_Render(t *testing.T) {
```

Comment:
Should we add test ensuring KubeadmConfig fields get translated into the proper ignition construct? 
(we are only testing the template renders, not if the template renders something correct)

Reply:
True, I'll improve that.

Reply:
Done. Now we parse created data as Ignition to make sure it's correct. We don't do strict parsing, as it seems used version of `ct` generates Ignition, which triggers some warnings. This can probably be improved if or when Flatcar updates to newer version of Ignition.